swagger-spec/addons/definition.yaml

```diff
@@ -0,0 +1,36 @@
+type: object
+title: Addon
+properties:
+  links: {}
+  attributes:
+    type: object
+    title: Attributes
+    readOnly: false
+    description: The properties of the addon entity.
+    properties:
+      url:
+        type: string
+        readOnly: true
+        description: Url to the third-party service.
+      name:
+        type: string
+        readOnly: true
+        description: Full name of the third-party service provider.
+      description:
+        type: string
+        readOnly: true
+        description: Description of the addon.
+      categories:
+        type: array
+        items:
+          type: string
```

Comment:
Can you incorporate the `enum_fields` key here? It looks like addon categories are listed [here](https://github.com/CenterForOpenScience/osf.io/blob/develop/website/settings/defaults.py#L235).